src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -182,6 +183,15 @@
       </IncludedFileForRunnerScript>
     </ItemGroup>
     
+    <Copy Condition="'$(TestAppX)' == 'true'"
```

Comment:
We shouldn't need a new runner template we should be able to use the same template just point at a different host and add these additional files to the copy list.